### PR TITLE
Add recipe for sarge

### PR DESCRIPTION
**Pull request by takluyver**
We need sarge for some domain specific code we're hoping to use conda-forge to package.

### CODE REVIEW

File: recipes/sarge/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+test:
+  imports:
+    - sarge
+
+about:
+  home: http://sarge.readthedocs.io/en/latest/
+  license: BSD 3-clause
+  license_family: BSD
+  summary: a wrapper for subprocess for interaction with external applications
+  license_file: LICENSE
```
Comment (synapticarbors) [inline recipes/sarge/meta.yaml]: please add:

```
doc_url: http://sarge.readthedocs.io/en/latest/
dev_url: https://bitbucket.org/vinay.sajip/sarge
```

File: recipes/sarge/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: pip install --no-deps .
```
Comment (synapticarbors) [inline recipes/sarge/meta.yaml]: Please install via `python setup.py install`

File: recipes/sarge/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+requirements:
+  build:
+    - python
+    - pip
```
Comment (synapticarbors) [inline recipes/sarge/meta.yaml]: If installing via `setup.py` remove `pip`.

File: recipes/sarge/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  home: http://sarge.readthedocs.io/en/latest/
+  license: BSD 3-clause
+  license_family: BSD
+  summary: a wrapper for subprocess for interaction with external applications
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - takluyver
+    - fangohr
```
Comment (synapticarbors) [inline recipes/sarge/meta.yaml]: @fangohr, @mb1a15, @davidcortesortuno are you ok being added as co-maintainers of the `sarge` package on conda-forge?

File: recipes/sarge/meta.yaml (hunk 2 above)
Comment (takluyver) [inline recipes/sarge/meta.yaml]: Is there a reason to prefer that over pip? AIUI using pip is always recommended now to install packages. This is also what we do [for the ipython feedstock](https://github.com/conda-forge/ipython-feedstock/blob/8121ecd5043b96fd7fe9f075a56ef09eaf016995/recipe/meta.yaml#L14), among a number of other recipes.

Comment (takluyver) [inline recipes/sarge/meta.yaml]: Done.

File: recipes/sarge/meta.yaml (hunk 1 above)
Comment (takluyver) [inline recipes/sarge/meta.yaml]: Done.

File: recipes/sarge/meta.yaml (hunk 4 above)
Comment (davidcortesortuno) [inline recipes/sarge/meta.yaml]: yes I am

File: recipes/sarge/meta.yaml (hunk 2 above)
Comment (synapticarbors) [inline recipes/sarge/meta.yaml]: I had not seen the ipython feedstock before and was mainly going off of my experience reviewing other recipes as well as what is explicitly mentioned in the conda-forge docs. I found the following discussion about what should be "standard practice":  https://github.com/conda-forge/staged-recipes/issues/528. The conclusion there was not definitive, but it seems like using `pip` is ok in the way that you did.

Comment (takluyver) [inline recipes/sarge/meta.yaml]: Thanks :-)

File: recipes/sarge/meta.yaml (hunk 4 above)
Comment (fangohr) [inline recipes/sarge/meta.yaml]: Yes. Thanks for processing this so swiftly.

**Pull request merged**